00new_packages.autobuild

```diff
@@ -13,5 +13,5 @@ end
 
 cmake_package 'gui/rock_webapp'
 ruby_package 'tools/rest_api' do |pkg|
-    Autoproj.env_set 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
+    Autoproj.env_set 'ROCK_WEBAPP', File.join(Autoproj.build_dir, 'share', 'rest_api')
```

Comment:
ROCK_WEBAPP defines the configuration directory so a single path variable is wanted.
Inside the configuration directory a "plugins" folder may esist, which is evaluated

I think you should use Autobuild.prefix instead of Autoproj.build_dir to fix this this config dir issue.

For the plugins one could think about adding a additional ROCK_WEBAPP_PLUGIN_PATH but i remember vaguely that for some reason we decided to do an "install" for the plugins.

Having this new plugin path would be fine for me. but this is unrelated to the faulty setting of the config dir


Autobuild.prefix